docs/platform/linode-gpu/getting-started-with-gpu/index.md

```diff
@@ -113,26 +143,46 @@ Installation on CentOS and RHEL operating systems will fail following this as th
 +-----------------------------------------------------------------------------+
 {{< /output >}}
 
-    In the output above, we can see that our Driver is installed and functioning correctly, the version of CUDA attributed to it, and other useful statistics.
+    In the output above, we can see that our driver is installed and functioning correctly, the version of CUDA attributed to it, and other useful statistics.
 
 ## Manual Install
 
-This section will walk you through the process of downloading and installing the latest NVIDIA driver on Debian 9. This process can be completed similarly on a Distro of your choice if needed:
+This section will walk you through the process of downloading and installing the latest NVIDIA driver on Debian 9. This process can also be completed similarly on another distro of your choice if needed:
+
+1. Visit NVIDIA's [Driver Downloads Page](https://www.nvidia.com/Download/index.aspx?lang=en-us).
+
+1. Make sure that the options from the drop-down menus reflect the following values:
 
-1. Find the latest Driver from NVIDIA's [Driver Downloads Page](https://www.nvidia.com/Download/index.aspx?lang=en-us).
+    | Prompt | Selection |
+    |--------|-----------|
+    | Product Type | Quadro |
+    | Product Series | Quadro RTX Series |
+    | Product | Quadro RTX 8000 |
+    | Operating System | Linux 64-bit |
+    | Download Type | Optimal Driver for Enterprise (ODE) |
```

Comment:
I actually couldn't get this second Download Type option to appear in their form. Can you show me where you saw this? I only saw one listing for the Download Type in the form, and the only option available for it was Linux Long Lived Driver

Reply:
This is a bug involving a cached response after searching for the driver. After clicking "search", go back to the downloads page and you should see the second Download Type, along with your previously chosen options pre-selected.